app/src/main/java/com/owncloud/android/authentication/AuthenticatorActivity.java

```diff
@@ -311,6 +312,11 @@ protected void onCreate(Bundle savedInstanceState) {
         if (getIntent().getBooleanExtra(EXTRA_USE_PROVIDER_AS_WEBLOGIN, false)) {
             webViewLoginMethod = true;
             webloginUrl = getString(R.string.provider_registration_server);
+        }
+        //enabling login url input screen in debug mode
+        //this will be useful in switching the environments during testing
```

Comment:
If this is needed I would put it in src/debug, so that this can never be in "gplay"/release build variant.

Reply:
Moved to debug variant.